cmd/channel/distributed/receiver/main.go

```diff
@@ -103,8 +111,13 @@ func main() {
 	}
 	defer kafkaProducer.Close()
 
+	var env envConfig
+	if err := envconfig.Process("", &env); err != nil {
+		logger.Fatal("Failed To Initialize env config", zap.Error(err))
+	}
+	channelReporter := eventingchannel.NewStatsReporter(env.ContainerName, kmeta.ChildName(env.PodName, uuid.New().String()))
```

Comment:
The "distributed" channel implementation already has its own environment loading implementation.  While the `kelseyhightower/envconfig` implementation might be worth looking at in terms of replacing/refactoring the current implementation, I'd rather not just bolt-on another pathway for handling environment variables.  Can we add the `POD_NAME` and `CONTAINER_NAME` to the existing `pkg/channel/distributed/reciever/env` logic instead?

Reply:
Sure I will update.